typescript/packages/browser/package.json

```diff
@@ -1,41 +1,34 @@
 {
     "name": "@0x/mesh-browser",
-    "version": "1.0.0",
+    "version": "8.2.0",
     "description": "TypeScript and JavaScript bindings for running Mesh directly in the browser.",
     "main": "./lib/index.js",
     "license": "Apache-2.0",
     "scripts": {
-        "prettier": "prettier --write '**/*.{ts,tsx,json,md}' --config .prettierrc",
         "build": "yarn build:go && yarn build:generate && yarn build:ts && yarn build:bundle",
-        "build:bundle": "node --max_old_space_size=3072 ./node_modules/webpack/bin/webpack.js --mode=development",
-        "build:ts": "node --max_old_space_size=4096 ./node_modules/.bin/tsc -b",
-        "build:generate": "INPUT_PATH=./wasm/main.wasm OUTPUT_PATH=./ts/generated/wasm_buffer.ts go run ./scripts/generate_wasm_buffer.go",
+        "build:bundle": "node --max_old_space_size=3072 ./node_modules/.bin/webpack --mode=development",
+        "build:ts": "tsc -b",
+        "watch:ts": "tsc -b -w",
```

Comment:
Hm, in the monorepo we only have these commands in the top-level package.json since they partially build/watch all packages. Does it work to call them from within a specific package's dir? If so, we _could_ keep it here, but we should def. have them in the monorepo root package.json.

Reply:
It actually does work when calling from a single package. That actually sounds kind of useful, but I'll add a top level command that doesn't use `wsrun` to make sure that it actually watches the whole repo.